src/Line3d.ts

```diff
@@ -26,13 +25,13 @@ export default class Line3d implements IDrawableObject {
     geometry.setPositions(new Float32Array(this.bufferSize));
     const material = new LineMaterial({ color: "#f00", linewidth: 2, worldUnits: false });
     this.lineMesh = new LineSegments2(geometry, material);
-    this.lineMesh.layers.set(MESH_LAYER);
+    this.lineMesh.layers.set(MESH_NO_PICK_OCCLUSION_LAYER);
```

Comment:
I moved the line meshes from the mesh layer to a new layer. This is because:
1. Lines need to write depth information so they interact with the volume (so the lines appear to fade into the volume if they intersect)
2. BUT, lines shouldn't interact with the pick buffer, since it can cause unexpected outlining behavior as below:

<img width="300" height="306" alt="Image" src="https://github.com/user-attachments/assets/00d02eeb-7193-44cf-8a3e-c31729379ae4" />

Now, the lines are rendered after the pick buffer is filled in but before volumes are rendered. There are other approaches to solving this, of course, but this was a low-lift solution that doesn't increase our memory requirements.